### PR TITLE
Build: remove unneeded references to jacoco

### PR DESCRIPTION
**Pull request by snazy**
After this change, IntelliJ's "Run with Coverage" still works.

### CODE REVIEW

**Comment by XN137** [comment]
isnt the idea of jacoco to produce a code coverage report as part of CI ?
are you saying that is no longer needed as people can do it with intellij if they want?

**Comment by snazy** [comment]
> isnt the idea of jacoco to produce a code coverage report as part of CI ? are you saying that is no longer needed as people can do it with intellij if they want?

The generated reports weren't used by anybody, because IJ generates better and "better focused" reports.

File: tools/server-admin/build.gradle.kts [hunk 1]
```diff
@@ -164,6 +163,8 @@ listOf("checkstyleTest", "compileTestJava").forEach { name ->
   tasks.named(name).configure { dependsOn(tasks.named("compileQuarkusTestGeneratedSourcesJava")) }
 }
 
+tasks.named("quarkusDependenciesBuild").configure { dependsOn("processJandexIndex") }
```
Comment (XN137) [inline tools/server-admin/build.gradle.kts:166]: intentionally included?

**Review by XN137 — APPROVED** [review]
> The generated reports weren't used by anybody, because IJ generates better and "better focused" reports.

for me an overall CI code coverage report sounds useful but i trust you know the main developer usecases better than i do.

changes lgtm

File: tools/server-admin/build.gradle.kts (hunk 1 above)
Comment (snazy) [reply tools/server-admin/build.gradle.kts:166]: yea - task dependency mess

**Comment by snazy** [comment]
> for me an overall CI code coverage report sounds useful but i trust you know the main developer usecases better than i do.

We had that for some time - but the results were just wrong. The tool wasn't able to identify parameterized test runs (in any way), and got completely mad with the spark extensions.

**Comment by snazy** [comment]
Oh - and nobody ever looked at the results :)

**Pull request merged**